LSLib/Granny/Model/ColladaImporter.cs

```diff
@@ -729,6 +738,8 @@ private void ImportSkin(Root root, skin skin)
 
                     offset += stride;
                 }
+
+                Debug.Assert(total == 0 || total == 255);
```

Comment:
Added this as a sanity check since I saw other spots using `Debug.Assert`. Happy to remove it if it doesn't make sense here.